ED/src/dynamics/rk4_derivs.F90

```diff
@@ -1821,6 +1896,13 @@ subroutine canopy_derivs_two(mzg,initp,dinitp,csite,ipa,hflxsc,wflxsc,qwflxsc,hf
          !---------------------------------------------------------------------------------!
          dinitp%wood_energy(ico) = 0.d0
          dinitp%wood_water(ico)  = 0.d0
+         dinitp%wood_hcap(ico)   = 0.d0
```

Comment:
We discussed this offline, and this won't affect the legacy simulations.  But just to keep it registered, heat capacity should not be a prognostic variable, ideally internal water mass should be the tracked variable with the differential equation, then heat capacity could be updated diagnostically.